mindsdb/integrations/handlers/documentdb_handler/Manual_QA.md

```diff
@@ -0,0 +1,86 @@
+# Welcome to the MindsDB Manual QA Testing for DocumentDB Handler
```

Comment:
Let's exclude Manual QA file, we are removing this files.

Reply:
Removed the Manual QA